src/Classes/ImportTab.lua

```diff
@@ -1138,6 +1139,20 @@ function HexToChar(x)
 	return string.char(tonumber(x, 16))
 end
 
+function ReplaceDiscrimintorSafely(accountName)
+	reversedAccountName = string.reverse(accountName)
+	discriminatorIndex = 0
+	for i = 1, #reversedAccountName do
+		local c = reversedAccountName:sub(i,i)
+			if c == "#" or c == "-" then
+			discriminatorIndex = i
+			break
+			end
+	end
+	discriminatorIndex = discriminatorIndex * -1
```

Comment:
FYI, you don't have to reverse the string and the index here, just put your for loop in reverse instead
```suggestion
	discriminatorIndex = 0
	for i = #accountName, 1  do
		local c = accountName:sub(i,i)
			if c == "#" or c == "-" then
			discriminatorIndex = i
			break
			end
	end
```

Reply:
Didn't know that was possible in lua, cool stuff, but I see you removed it anyways.

`gsub` not working with regex totally did not click in my brain, props for finding a better shorthand.